web_responsive/static/src/css/web_responsive.scss

```diff
@@ -181,23 +177,31 @@ $chatter_zone_width: 35%;
 
         @include media-breakpoint-up(lg) {
             padding: {
-                left: 20vw;
-                right: 20vw;
+                left: calc((100vw - 850px) / 2);
+                right: calc((100vw - 850px) / 2);
             }
         }
 
         .o_app {
             align-items: center;
             display: flex;
+            padding: 10px 0;
             flex-direction: column;
             justify-content: flex-start;
             background: none;
+            transition: all 0.3s ease 0s;
```

Comment:
Remember that `transition: all` is dangerous. Could you be more specific?

Reply:
Done, thanks!